omnigibson/prims/entity_prim.py

```diff
@@ -1160,7 +1160,8 @@ def kinematic_only(self, val):
                 body's pose / velocities. See https://docs.omniverse.nvidia.com/app_create/prod_extensions/ext_physics/rigid-bodies.html?highlight=rigid%20body%20enabled#kinematic-rigid-bodies
                 for more information
         """
-        self.root_link.kinematic_only = val
+        for link in self.links.values():
+            link.kinematic_only = val
```

Comment:
This is handling the case where the object has fixed joints?

Reply:
yep, exactly!

Reply:
this was causing the code to break when loading a normal scene